content/docs/start/visualization.md

```diff
@@ -0,0 +1,113 @@
+# Visualization with Plots
+
+In this section, we'll add visualization to the [`example-dvc-experiments`][ede]
+project. If you would like to try these yourself, please refer to the project
+[README] about how to install.
+
+[ede]: https://github.com/iterative/example-dvc-experiments
+[readme]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/README.md
+
+### Creating plots from text with DVC
+
+A useful plot to show the classification performance is the confusion matrix. In
+order to produce it, DVC expects a CSV **plot file** in the form:
+
+```csv
+actual,predicted
+0,0
+0,2
+...
+```
+
+> We added a [loop] comparing the results to generate this file from the
+> predictions.
+
+[loop]:
+  https://github.com/iterative/example-dvc-experiments/blob/main/src/train.py#L123
+
+After running the experiment with `dvc exp run`, DVC will produce
+`plots/confusion.csv` file presenting the classifier performance and open it in
+the browser:
```

Comment:
Confusing to say that a CSV file presents something. `dvc plots` will present the CSV file actually, right? Maybe

```suggestion
Running the experiment with `dvc exp run` will produce
`plots/confusion.csv`. Use `dvc plots show` to present it as an HTML
file, and open it in the browser:
```